### PR TITLE
Implement reviews

### PR DESCRIPTION
**Pull request by tclaus**
A guest can send a review some days after a event was requested. 
Regardless if the event actually was performed oder just a request. 

The host should be reviewed for its communication and how the experience was to make this event happen

### CODE REVIEW

File: app/controllers/location_reviews_controller.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        render 'reviews/_review_form'
+      else
+        # Maybe show the review
+        logger.info " A review for this token was already set #{token}"
+        flash[:alert] = t('.a_review_already_exists')
+        redirect_to(location_path(@reservation.location)) && return
+      end
+    else
+      logger.info " Invalid review token #{token}"
+      flash[:alert] = t('.invalid_reservation_token')
```
Comment (ThorstenClaus) [inline app/controllers/location_reviews_controller.rb:21]: Flash an log message do not match

File: app/assets/javascripts/review-raty.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+$(document).ready(function() {
+  $('#stars').raty({
+      path: '/assets',
```
Comment (ThorstenClaus) [inline app/assets/javascripts/review-raty.js:4]: Do the assets (star-icons) show on site?

File: app/jobs/send_review_reminder_job.rb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  def send_review_reminder
+    # Find reservations without an review
+    # And with a start_date which is in the past
+    reservations_without_review = Reservation.not_invited_for_review
+    if reservations_without_review.empty?
+      logger.info 'No review invitations to send'
+    end
+    # What if, ssomeone sends masses of requests, just to can sent a review?
+
+    # Last diff(start_date ) > 5 Tage
```
Comment (ThorstenClaus) [inline app/jobs/send_review_reminder_job.rb:23]: Remove these comments

File: app/controllers/location_reviews_controller.rb (hunk 1 above)
Comment (tclaus) [reply app/controllers/location_reviews_controller.rb:21]: Fixed log message. The token is a reservation token, but it is used to enable a individual review for that reservation

File: app/assets/javascripts/review-raty.js (hunk 2 above)
Comment (tclaus) [reply app/assets/javascripts/review-raty.js]: No it doesn't. On Heroku the /assets folder is compiled. Have moved this static files to the /public/assets

File: app/jobs/send_review_reminder_job.rb (hunk 3 above)
Comment (tclaus) [reply app/jobs/send_review_reminder_job.rb:23]: Removed

**Pull request merged**